app/template/[templateId]/q/new/page.tsx

```diff
@@ -2,51 +2,150 @@
 
 import React from 'react';
 import {
-  Button,
-  FieldError,
-  Form,
-  Input,
-  Label,
-  Text,
-  TextField
+    Breadcrumb,
+    Breadcrumbs,
+    Button,
+    FieldError,
+    Input,
+    Label,
+    Link,
+    SearchField,
+    Text
 } from "react-aria-components";
+import PageHeader from "@/components/PageHeader";
+import {Card, CardBody, CardFooter, CardHeading} from "@/components/Card/card";
 
-const QuestionCreatePage: React.FC = () => {
-  return (
-    <>
-      <h1>Create a New Question</h1>
+const QuestionTypeSelectPage: React.FC = () => {
 
-      <Form>
-        <TextField
-          name="question_text"
-          type="text"
-          isRequired
-        >
-          <Label>Question text</Label>
-          <Text slot="description" className="help">
-            Enter the question that you want to include in this section.
-          </Text>
-          <Input/>
-          <FieldError/>
-        </TextField>
+    const templates = [
+        {
+            title: 'Long text question',
+            link: '/template/353535/q/q_mnopqr',
+            text: 'For questions that require long answers, you can select formatting options too.',
+            type: 'Text',
+        },
+        {
+            title: 'Short text question',
+            link: '/template/353535/q/q_mnopqr',
+            text: 'For questions that require short, simple answers.',
+            type: 'Text',
+        },
+        {
+            title: 'URL field',
+            link: '/template/353535/q/q_mnopqr',
+            text: 'When you need the user to supply a URL. Includes validation for proper URL format.',
+            type: 'URL',
+        },
+        {
+            title: 'Radio selection',
+            link: '/template/353535/q/q_mnopqr',
+            text: 'For simple questions with only valid answer. Perfect for multiple choice questions.',
+            type: 'Choice',
+        },
+        {
+            title: 'Checkbox group',
+            link: '/template/353535/q/q_mnopqr',
+            text: 'For questions with multiple valid answers. Allows users to select multiple options.',
+            type: 'Choice',
+        }];
 
-        <TextField
-          name="question_help_text"
-          type="text"
-        >
-          <Label>Help text</Label>
-          <Text slot="description" className="help">
-            Optionally, provide help text or additional context for this question.
-          </Text>
-          <Input/>
-          <FieldError/>
-        </TextField>
+    const templates_others = [
+        {
+            title: 'ORCID search',
+            link: '/template/353535/q/q_mnopqr',
```

Comment:
Is it intentional that the "h1" heading is smaller than the "h2" heading on this page: 
<img width="1171" alt="image" src="https://github.com/user-attachments/assets/aec506be-723a-41ed-b983-d2bc783a7077">

Reply:
Mistake - will fix